changes/2076.changed

```diff
@@ -0,0 +1 @@
+Changed the `created` field of all models from a DateField to a DateTimeField for added granularity. Preexisting records will show as created at midnight UTC on their original creation date.
```

Comment:
Nit: Is it actually midnight UTC or is it midnight in the database's TZ?

Reply:
Fair question! Looks to be UTC:

```python
>>> settings.TIME_ZONE
'America/Chicago'
>>> Site.objects.first().created
datetime.datetime(2023, 2, 15, 0, 0, tzinfo=<UTC>)
```

![image](https://user-images.githubusercontent.com/5603551/219182463-b4c9bcb3-811c-4d61-bb78-8baba86d1e95.png)

Reply:
Thanks for checking!